tests/openwisp2/sample_config/migrations/0009_organizationconfigsettings_approximate_location_enabled_and_more.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 5.2.1 on 2025-07-11 16:28
+
+from django.db import migrations, models
+
+import openwisp_utils.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("sample_config", "0008_whoisinfo_organizationconfigsettings_whois_enabled"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="organizationconfigsettings",
+            name="approximate_location_enabled",
+            field=openwisp_utils.fields.FallbackBooleanChoiceField(
+                blank=True,
+                default=None,
+                fallback=False,
+                help_text="Whether the approximate location feature is enabled",
+                null=True,
+                verbose_name="Approximate Location Enabled",
+            ),
+        ),
+        migrations.AddField(
+            model_name="whoisinfo",
+            name="latitude",
+            field=models.FloatField(blank=True, help_text="Latitude", null=True),
```

Comment:
Do we any validation checks in the code to ensure that latitude and longitude values are correct?